lit_gpt/config.py

```diff
@@ -296,6 +296,24 @@ def norm_class(self) -> Type:
         _mlp_class="LLaMAMLP",
         intermediate_size=13824,
     ),
+    # https://huggingface.co/TheBloke/Nous-Hermes-13B/blob/main/config.json
+    dict(
+        org="NousResearch",
+        name="Nous-Hermes-13b",
+        block_size=2048,
+        vocab_size=32001,
+        padding_multiple=1,
```

Comment:
Since there's no padding, we can just fix the "final" vocab size

```suggestion
        padded_vocab_size=32001,
```